docs/platforms/apple/common/configuration/app-hangs.mdx

```diff
@@ -7,6 +7,9 @@ description: "Learn about how to add app hang detection reporting."
 <Note>
 
 We recommend disabling this feature for Widgets and Live Activities because it might detect false positives.
+Furthermore, the Cocoa SDK might report app hangs when the OS opens a system dialog asking for specific permissions,
```

Comment:
```suggestion

Furthermore, the Cocoa SDK might report app hangs when the OS opens a system dialog asking for specific permissions,
```
I suggest starting a new paragraph here, since the recommendation for this scenario is different.